docs/collecting-data/code-generation/index.md

```diff
@@ -8,14 +8,10 @@ sidebar_label: "Code Generation"
 This documentation only applies to [Snowplow BDP](https://snowplow.io/snowplow-bdp/). See the [feature comparison](/docs/feature-comparison/index.md) page for more information about the different Snowplow offerings.
 :::
 
-Snowplow has created tooling around assisting development efforts on instrumenting, verifying and maintaining tracking code that works with Snowplow tracking SDKs. This is the `Snowtype CLI`.
+The `Snowtype CLI` is a tool that assists with instrumenting, verifying and maintaining tracking code that works with Snowplow tracking SDKs. 
 
 ```mdx-code-block
 import DocCardList from '@theme/DocCardList';
 
 <DocCardList />
 ```
-
-## Getting help or giving feedback
```

Comment:
@igneel64 John makes a good point that Discourse might not be the best platform, seeing as how this is a BDP-specific doc. But do we have a different channel for feedback?